<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
     MIDI Virtual Piano Keyboard
     Copyright (C) 2008-2018, Pedro Lopez-Cabanillas

     This program is free software; you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
     the Free Software Foundation; either version 3 of the License, or
     (at your option) any later version.

     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.

     You should have received a copy of the GNU General Public License along
     with this program; If not, see http://www.gnu.org/licenses/
 </comment>
 <class>NetworkSettingsDialog</class>
 <widget class="QDialog" name="NetworkSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Driver Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkIPv6">
     <property name="text">
      <string>Use IPv6</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInterface">
     <property name="text">
      <string>Network Interface:</string>
     </property>
     <property name="buddy">
      <cstring>comboInterface</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboInterface"/>
   </item>
   <item>
    <widget class="QLabel" name="lblAddress">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtAddress"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetworkSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetworkSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
